### ISSUE TITLE
Python 3: Add Cover Image hangs

### ISSUE THREAD

**Issue by cclauss** [internetarchive/openlibrary#3687]
### Evidence / Screenshot (if possible)
http://localhost:8080/works/OL6037022W/Remix?debug=true on Docker Python 3
* Or http://staging.openlibrary.org/works/OL6037022W/Remix?debug=true

Command to launch Open Library on Docker on Python 3:
```
docker-compose down ; \
    PYENV_VERSION=3.9.0 docker-compose -f docker-compose.yml -f docker-compose.infogami-local.yml up -d ; \
    docker-compose logs -f --tail=10 web
```
### Relevant url?

1. http://localhost:8080/works/OL6037022W/Remix?debug=true
2. On the image, click Add Cover Image
3. Browse and select an appropriate local image file and click Submit
4. Internal Server Error

### Steps to Reproduce

1. Go to ...
2. Do ...

* Actual: 
* Expected: 

### Details

- **Logged in (Y/N)?** 
- **Browser type/version?** 
- **Operating system?** 
- **Environment (prod/dev/local)?** prod

**Comment by cclauss** [comment]
I am struggling with how to debug this on
http://localhost:8080/books/OL23703735M/Remix/add-cover?debug=true
1. The problem occurs but the debug=true does not provide the internal debugger.
2. The VS Code debugger is not stopping on the breakpoints that I have set.
3. If I use Python `breakpoint()` then `docker-compose up` fails.
4. Looking at the logs inside Docker `cat /var/log/openlibrary/ol-errors/2020-08-12/* | more` provides more heat than light.
5. I was able to use Firefox Inspector to get to this but I do not speak JavaScript...

<img width="910" alt="Screenshot 2020-08-12 at 18 04 21" src="https://user-images.githubusercontent.com/3709715/90039298-bbc35f00-dcc6-11ea-8701-d5ee37e73583.png">
